src/conf.c

```diff
@@ -199,8 +199,8 @@ struct {
         STDCONF (defaulterrorfile, STR, handle_defaulterrorfile),
         STDCONF (statfile, STR, handle_statfile),
         STDCONF (stathost, STR, handle_stathost),
-        STDCONF (xtinyproxy,  BOOL, handle_xtinyproxy),
         /* boolean arguments */
```

Comment:
I saw the comment out of place when reviewing these regexes for the documentation about quoting. But this change can be reverted if we want to keep the PR strictly in the docs directory.

Reply:
yes, please move this to separate PR; it's unrelated to what this PR does.

Reply:
Reverted conf.c change in a806636